app/code/core/Mage/Admin/sql/maho_setup/maho-24.12.1.php

```diff
@@ -11,12 +11,12 @@
 
 /** @var Mage_Checkout_Model_Resource_Setup $this */
 $installer = $this;
-
 $installer->startSetup();
 
 $installer->run("
-    ALTER TABLE {$this->getTable('sales/order_status')}
-    ADD COLUMN `color` VARCHAR(20) NULL
+    ALTER TABLE {$this->getTable('admin/user')}
+    ADD COLUMN `passkey_credential_id_hash` varchar(255) NULL,
+    ADD COLUMN `passkey_public_key` VARCHAR(255) NULL
 ");
```

Comment:
Was this left over code from testing?

Reply:
It got broken from a `git stash` situation 😅 now it should be ok